packages/unmock-core/src/nock.ts

```diff
@@ -215,15 +215,15 @@ type InputToPoet = { [k: string]: any } | Primitives | Primitives[];
 
 // How the fluent dynamic service API looks like (e.g. specifies `get(endpoint: string) => DynamicServiceSpec`)
 export interface IFluentDynamicService {
-  tldr: () => void;
-  get: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  head: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  post: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  put: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  patch: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  delete: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  options: (endpoint: ValidEndpointType) => DynamicServiceSpec;
-  trace: (endpoint: ValidEndpointType) => DynamicServiceSpec;
+  tldr(): void;
```

Comment:
Better described without arrow notation, as this way these are declared as methods; with arrow notation, they're declared as properties/attributes.